app/src/main/java/com/github/lookupgroup27/lookup/ui/overview/Landing.kt

```diff
@@ -1,5 +1,79 @@
-package com.github.lookupgroup27.lookup.ui.overview
```

Comment:
We need the package to be specified

Reply:
Thanks for pointing that out! I'll add the package declaration.